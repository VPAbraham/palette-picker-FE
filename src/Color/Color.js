import React from 'react';
import './Color.scss';

export const Color = (props) => {
  const style = {
    background: props.color
  }
  return (
    <div className={props.position} style={style} id="color">
    </div>
  )
}

export default Color;
