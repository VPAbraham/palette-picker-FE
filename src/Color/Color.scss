#color {
  height: 23vh;
  width: 23vh;
  border-radius: 1px;

  &:hover {
    
  }
}

.color1 {
  grid-area: 1 / 2 / 2 / 3;
  background: green;

}

.color2 {
  grid-area: 2 / 1 / 3 / 2;
  background: red;

}

.color3 {
  grid-area: 2 / 2 / 3 / 3;
  background: blue;

}

.color4 {
  grid-area: 2 / 3 / 3 / 4;
  background: purple;

}

.color5 {
  grid-area: 3 / 2 / 4 / 3;
  background: salmon;

}