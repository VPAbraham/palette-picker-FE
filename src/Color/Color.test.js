import React from 'react';
import { Color } from './Color';
import { shallow } from 'enzyme';

describe('Color', () => {
  let wrapper;
  beforeEach(() => {
    wrapper = shallow(<Color
      position={'color1'}
      color={'#baddc3'}
    />)
  });
  it('should match the snapshot with all data passed in', () => {
    expect(wrapper).toMatchSnapshot();
  });
});