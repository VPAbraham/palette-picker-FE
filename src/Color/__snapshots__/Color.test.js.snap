// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Color should match the snapshot with all data passed in 1`] = `
<div
  className="color1"
  id="color"
  style={
    Object {
      "background": "#baddc3",
    }
  }
/>
`;
